src/ioctls/vcpu.rs

```diff
@@ -1943,6 +1950,16 @@ mod tests {
                     let vcpu_regs = vcpu_fd.get_regs().unwrap();
                     assert_eq!(vcpu_regs.rip, expected_rips[instr_idx]);
                     instr_idx += 1;
+                    assert_eq!(
+                        debug,
+                        kvm_debug_exit_arch {
+                            exception: 1,
+                            pad: 0,
+                            pc: vcpu_regs.rip,
+                            dr6: 0b11111111111111110100111111110000,
+                            dr7: 1 << 10,
```

Comment:
What's the meaning of these? I just ran the tests on my local machine and this one fails with:
```rust
---- ioctls::vcpu::tests::test_run_code stdout ----
thread 'ioctls::vcpu::tests::test_run_code' panicked at 'assertion failed: `(left == right)`
  left: `kvm_debug_exit_arch { exception: 1, pad: 0, pc: 4099, dr6: 4294856688, dr7: 1024 }`,
 right: `kvm_debug_exit_arch { exception: 1, pad: 0, pc: 4099, dr6: 4294922224, dr7: 1024 }`'
```

Reply:
Good catch! I just pinned these to the values on my machine. On your machine, the RTM (restricted transactional memory) flag (bit 16) in DR6 is not set in this case. “This bit is always 1 if
the processor does not support RTM.” (Intel Manual Volume 3B, 17.2.3)

I'll rework the test case to not check this bit and also not check any reserved bits.

Reply:
I adjusted the test. Could you check again, please? :)

Reply:
Thanks!